XamarinCommunityToolkit/Views/BaseTemplatedView.shared.cs

```diff
@@ -0,0 +1,33 @@
+using System.ComponentModel;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Any reason to hide this for the lib users? I see this as a great helper if any dev. needs to create a templated view control

Reply:
I don't want that this control appears as a hint in XAML. 
Also, usually, users inherit custom controls directly from "TControl" class

e.g.
```
class Expander : StackLayout {
}
```

Reply:
I see, so can we make this `internal` instead of using the `EditorBrowsableState.Never`?

Reply:
No, we can't inherit **public** class from **internal**.

Reply:
Hm... That makes sense and this also explains why we have this attribute in other places, tks.